CHANGELOG.md

```diff
@@ -4,6 +4,23 @@
 <!--- Distributed under the Boost Software License, Version 1.0. (See accompanying -->
 <!--- file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt) -->
 
+## 0.17.0 (2023-08-02)
+
+### New features
+
+- Add `pika::wait` to wait for the runtime to be idle. ([#704](https://github.com/pika-org/pika/pull/704))
+- The debugger can now be attached after printing failure information. Previously the debugger could be attached before printing the error information. ([#712](https://github.com/pika-org/pika/pull/712))
+- `handle_print_bind` now also prints the thread pool when a mask is empty to be consistent to the printing when a mask is set. ([#710](https://github.com/pika-org/pika/pull/710))
+- Allow to explicitly reset `{unique_,}any_sender`.  ([#719](https://github.com/pika-org/pika/pull/719))
+- Add `execution::unpack` sender adapter to unpack tuple-like types sent by predecessor senders. ([#721](https://github.com/pika-org/pika/pull/721))
```

Comment:
```suggestion
- Add `execution::unpack` sender adaptor to unpack tuple-like types sent by predecessor senders. ([#721](https://github.com/pika-org/pika/pull/721))
```
only because we've used adaptor over adapter almost excusively so far, but not necessarily because it's better or more correct.